BUILDER_CONTAINER.md

```diff
@@ -0,0 +1,39 @@
+# Builder Docker Container
+
+> *Warning: NOT INTENDED FOR PRODUCTION. USE AT OWN RISK.*
+
+## Building
+
+* `APP_HOSTNAME` - Builder's addressable hostname. This is used for redirection back after authenticating with GitHub and proxying requests to the appropriate http gateway _(default: "localhost")_.
+* `GITHUB_API_URL` - external GitHub or GitHub Enterprise API endpoint Builder will connect to _(default: "https://api.github.com")_.
+* `GITHUB_WEB_URL` - external GitHub or GitHub Enterprise web URL endpoint Builder will connect to _(default: "https://github.com")_
+* `GITHUB_CLIENT_ID` - GitHub OAuth application client-id *required*.
+* `GITHUB_CLIENT_SECRET` - GitHub OAuth application client-secret *required*.
+* `GITHUB_ADMIN_TEAM` - GitHub Team ID to grant admin gateway access to *required*.
+* `GITHUB_WORKER_TEAM` - GitHub Team ID to grant worker publishing access to *required*.
+* `WORKER_AUTH_TOKEN` - GitHub personal access token for authenticating worker publish requests *required*.
+
+```bash
+$ cd ${root}
+$ docker build --no-cache -f BLDR-Dockerfile --build-arg GITHUB_CLIENT_ID=02fb7d9e1fde99e8d395 --build-arg GITHUB_CLIENT_SECRET=8d7611da338fd741330804ec470236da58ffdd68 --build-arg GITHUB_API_URL=https://10.48.236.67/api/v3 --build-arg GITHUB_WEB_URL=https://10.48.236.67 --build-arg WORKER_AUTH_TOKEN=e96301212f26a8da99925d04fb5857e869792a47 -t habitat/builder .
+```
+
+## Running
+
+```
+$ docker run -p 80:80 -p 443:443 -p 9631:9631 -p 9638:9638 habitat/builder
+```
+
+## GitHub Self-Signed Certificates
+
+1. Retrieve the container ID of the running Builder container
+```
+$ docker ps
+```
+
+1. Import certificate
+```
+$ docker exec ${instance-id} hab pkg binlink core/openssl
+$ docker exec ${instance-id} openssl s_client -showcerts -connect ${SRV_ADDR}:443 </dev/null 2>/dev/null|openssl x509 -outform PEM >> /usr/local/share/ca-certificates/github.crt
```

Comment:
You could call `hab pkg exec core/openssl openssl s_client -showcerts -connect ${SRV_ADDR}:443 …` if you want to save on binlink'ing openssl.